Make deps fail in conda env

## What's this issue?

When using conda, running `make deps` fail. (It must be failing otherwise as well, didn't give it a try).


## How to reproduce it?
Here's some logs. Run `make deps` in a conda env.
```shell
(base) ➜  tach git:(main) ✗ make deps       
python -m pip install --upgrade uv
Requirement already satisfied: uv in /Users/girish/miniconda3/lib/python3.8/site-packages (0.2.25)

[notice] A new release of pip is available: 23.1.2 -> 24.1.2
[notice] To update, run: pip install --upgrade pip
Using Python 3.8.11 interpreter at: /Users/girish/miniconda3/bin/python3
Creating virtualenv at: .venv
Activate with: source .venv/bin/activate
Virtual environment created at .venv
source .venv/bin/activate && \
	uv pip install -r dev-requirements.txt
Resolved 48 packages in 398ms
Installed 48 packages in 57ms
 + annotated-types==0.7.0
 + backports-tarfile==1.2.0
 + build==1.2.1
 + certifi==2024.7.4
 + charset-normalizer==3.3.2
 + coverage==7.6.0
 + docutils==0.20.1
...
...
 + typing-extensions==4.12.2
 + urllib3==2.2.2
 + wcwidth==0.2.13
 + zipp==3.19.2
bash: line 1: maturin: command not found
make: *** [deps] Error 127
```

--- a/Makefile
+++ b/Makefile
@@ -28,6 +28,7 @@ deps: ## Install dependencies
 	uv pip install -r dev-requirements.txt
 
 	@unset CONDA_PREFIX && \
+	source $(VENV_BIN)/activate && \
 	maturin develop --profile release
 
 

